Dockerfile

```diff
@@ -23,12 +23,7 @@ RUN apt-get install -y nodejs
 
 RUN npm install
 RUN npm install babel-core@^7.0.0-bridge.0 --save-dev --dev jest --save-dev vue-jest vue-template-compiler --save-dev jest-serializer-vue -g yarn
-RUN npm run dev
+#RUN npm run dev
```

Comment:
There is a decent chance that this breaks Heroku.  We need to check that this builds in both environments before we close out the ticket for this.